cmd/helm/create.go

```diff
@@ -96,18 +112,15 @@ func (o *createOptions) run(out io.Writer) error {
 		AppVersion:  "0.1.0",
 		APIVersion:  chart.APIVersionV2,
 	}
-
-	if o.starter != "" {
-		// Create from the starter
-		lstarter := filepath.Join(o.starterDir, o.starter)
-		// If path is absolute, we don't want to prefix it with helm starters folder
-		if filepath.IsAbs(o.starter) {
-			lstarter = o.starter
+	if o.originalMeta {
+		chartYamlPath := filepath.Join(lstarter, "Chart.yaml")
+		originalChartfile, err := chartutil.LoadChartfile(chartYamlPath)
+		if err != nil {
+			return err
 		}
-		return chartutil.CreateFrom(cfile, filepath.Dir(o.name), lstarter)
+		originalChartfile.Name = chartname
+		cfile = originalChartfile
```

Comment:
Question: wouldn't this also preserves other information other than dependencies from `Chart.yaml`? In the PR description the main goal seems like to preserve dependencies, but I could be wrong.

Reply:
yes indeed, all the other information will be preserved)